content/00.front-matter.md

```diff
@@ -1,6 +1,5 @@
-# Manubot Rootstock: Manuscript Title
 
-## Authors
+# Authors
```

Comment:
In the resulting HTML, what heading level do these render to. If h1 and title is also h1, don't we want to keep these as level 2?